ddtrace/tracer/option.go

```diff
@@ -294,6 +294,11 @@ func SpanType(name string) StartSpanOption {
 	return Tag(ext.SpanType, name)
 }
 
+// MeasureSpan marks this span to be measured for metrics and stats calculations.
+func MeasureSpan() StartSpanOption {
```

Comment:
I'd probably have named this `Measured()`, so it's less stuttery in the places it's used.